src/Serializer/JsonEncoder.php

```diff
@@ -34,7 +34,8 @@ public function __construct(string $format, BaseJsonEncoder $jsonEncoder = null)
 
         // Encode <, >, ', &, and " for RFC4627-compliant JSON, which may also be embedded into HTML.
```

Comment:
This comment is outdated. See https://github.com/symfony/http-foundation/blob/99b9cc2044fe1d36f4c141da1496190cf5591ddf/JsonResponse.php#L30